modules/refine-testing/src/main/java/org/openrefine/runners/testing/RunnerTestBase.java

```diff
@@ -955,6 +954,31 @@ public void testLoadChangeDataDoesNotExist() throws IOException {
         SUT.loadChangeData(new File(tempDir, "doesNotExist"), stringSerializer);
     }
 
+    @DataProvider(name = "supportedCompressionFormats")
+    public static Object[][] supportedCompressionFormats() {
+        return new Object[][] {
+                { "gz" }, { "zst" }
+        };
```

Comment:
I've still kept the support for the previous compression format in tests, but maybe it's not necessary.